netmap/netmap_contract.go

```diff
@@ -162,11 +161,19 @@ func _deploy(data any, isUpdate bool) {
 			switchToNotary(ctx)
 		}
 
-		return
-	}
+		if args.version <= 19_000 {
+			balanceContract := storage.Get(ctx, balanceContractKey).(interop.Hash160)
+			key := append([]byte(newEpochSubscribersKey), append([]byte{byte(0)}, balanceContract...)...)
```

Comment:
You don't need `0` and `1` prefixes. Just use `newEpochSubscribersKey` plus contract hash, and it's exactly the unique key that you're looking for.

Reply:
https://github.com/nspcc-dev/neofs-contract/pull/368#discussion_r1402727722